app/controllers/chatops/slack_slash_command_controller.rb

```diff
@@ -0,0 +1,7 @@
+module Chatops
+  class SlackSlashCommandController < ::ApplicationChatopsController
+    def create
```

Comment:
**nitpick/suggestion**: I think it's fine in this PR since this is mostly setting the stage for the real slash command work, but as a pattern for future controllers, I recommend we avoid the awkwardness that is REST for webhooks and use a pattern of something like `#handle` for the actions. Each controller will only ever have a singular action and it'll operate in a lot of ways very differently than a traditional POST create action.

Nothing necessarily needs done here for this PR, and I'm happy to chat about it if you have thoughts.

Reply:
I'm fine with that change. I have no real hard opinion on this one. I'm happy to use `#handle` instead.